django/thunderstore/frontend/tests/test_errors.py

```diff
@@ -0,0 +1,24 @@
+import base64
+
+import pytest
+from django.conf import settings
+from django.test import RequestFactory
+
+from thunderstore.frontend.views import handle404, handle500
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize("static", (False, True))
+@pytest.mark.parametrize("view", (handle404, handle500))
+def test_frontend_error_cache_headers(rf: RequestFactory, static: bool, view):
+    # b64 to ensure no accidental overlap with setting value if not intended
+    path = (
+        settings.STATIC_URL
```

Comment:
Not a big issue (and perhaps this is only an issue inside my head), but when split to lines like this, for me this reads as JS-style ternary, i.e.
```
condition
value if True
value if False
```

Might be purely due to lack of exposure to Python recently.

Reply:
```python
value if condition is True else other_value
```
is how it looks on a single line, though here it was split to multiple thanks to black, but I can see how it can read weird after doing a bit of JS 😄